.github/workflows/docker.yml

```diff
@@ -0,0 +1,57 @@
+name: "build docker image"
+on:
+  push:
+    branches:
+      - publish-docker-image
+        #on:
+        #  schedule:
+        #    - cron:  '0 23 * * *'
+
+jobs:
+  build:
+    name: Build binary
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target: [x86_64-unknown-linux-gnu]
```

Comment:
No need to do a matrix or using cross if you are only targeting `x86_64-unknown-linux-gnu` just run a `cargo build`